File: introduction/model_answer/cpp/08_fourtune_cookies.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
	int A[4];
	for (int i = 0; i < 4; i++) {
		cin >> A[i];
	}
	sort(A, A+4);
	if (A[0] + A[3] == A[1] + A[2]) {
		cout << "Yes" << endl;
	} else if (A[0] + A[1] + A[2] == A[3]) {
		cout << "Yes" << endl;
	} else {
		cout << "No" << endl;
	}
	return 0;
}

File: introduction/model_answer/python/08_fourtune_cookies.py
# スペース区切りの入力を配列に格納するときはこのようにします
# 改行区切りでN個の整数を入力する場合は次のようにします
# [int(input()) for _ in range(N)]
A=list(map(int,input().split()))
A.sort()
if A[0]+A[3] == A[1]+A[2]:
	print('Yes')
elif A[0]+A[1]+A[2] == A[3]:
	print('Yes')
else:
	print('No')
